app/src/main/kotlin/com/flxrs/dankchat/utils/extensions/SpannableStringBuilderExtensions.kt

```diff
@@ -0,0 +1,26 @@
+package com.flxrs.dankchat.utils.extensions
+
+import android.content.Context
+import android.graphics.Typeface
+import android.text.SpannableStringBuilder
+import android.text.style.RelativeSizeSpan
+import android.text.style.StyleSpan
+import android.text.style.TextAppearanceSpan
+import android.text.style.TypefaceSpan
+import androidx.core.text.inSpans
+import com.flxrs.dankchat.R
+
+inline fun SpannableStringBuilder.timestampFont(
+    context: Context, // this is required just because we need to retreive the R.style stuff
+    builderAction: SpannableStringBuilder.() -> Unit
+): SpannableStringBuilder = inSpans(
+    TypefaceSpan("monospace"),
+    StyleSpan(Typeface.BOLD),
+    // style adjustments to make the monospaced text looks "same size" as the normal text
+    RelativeSizeSpan(0.95f),
```

Comment:
Extract this value to a constant please. I'm also fine with moving the function to the ChatAdapter file to have all the ui details there at one place